doc/reference/tooling/tarantoolctl.rst

```diff
@@ -1,8 +1,13 @@
 .. _tarantoolctl:
 
---------------------------------------------------------------------------------
-Utility `tarantoolctl`
---------------------------------------------------------------------------------
+Utility tarantoolctl (deprecated)
+=================================
+
+.. important::
+
+    ``tarantoolctl`` is deprecated in favor of :ref:`tt CLI <tt-cli>`.
+    Find the instructions on switching from ``tarantoolctl`` to ``tt`` in
+    :ref:`Migration from tarantoolctl to tt <tarantoolctl-migration-to-tt>`.
```

Comment:
> See also ``tarantoolctl`` usage examples in :ref:`Server administration <admin>`
section.

Am I correct that there are no more examples for tarantoolctl and this phrase can be removed?